open-sphere-base/core/src/main/java/io/opensphere/core/util/lang/ArithmeticUtilities.java

```diff
@@ -0,0 +1,177 @@
+package io.opensphere.core.util.lang;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.stream.Collectors;
+
+/**
+ * A collection of utilities to perform basic arithmetic operatoins on generic
+ * java.lang.Number instances.
+ */
+public final class ArithmeticUtilities
+{
+    /** Private constructor to prevent instantiation. */
+    private ArithmeticUtilities()
+    {
+        throw new UnsupportedOperationException("Instantiation of utility classes is not supported.");
+    }
+
+    /**
+     * Calculates the sum of the supplied numbers, and returns the result.
+     *
+     * @param leftNumber the left number portion of the sum.
+     * @param rightNumber the right number portion of the sum.
+     * @return the calculated sum.
+     */
+    public static Number sum(Number leftNumber, Number rightNumber)
```

Comment:
Could this take Number... as arguments instead?

Reply:
Done.